db/seeds/api.controller.js

```diff
@@ -1,8 +1,53 @@
 const endpointsJson = require("../../endpoints.json");
+const articles = require("../data/test-data/articles");
+const { checkArticleExists } = require("./api.articles.model");
+const { selectTopics, selectArticle } = require("./api.model");
 
-function getApi(req, res) {
+console.log("in controller");
+
+exports.getApi = (req, res) => {
   console.log(endpointsJson, "<--- endpoints json");
   res.status(200).send({ endpoint: endpointsJson });
-}
+};
+
+exports.getTopics = (req, res, next) => {
+  selectTopics()
+    .then((topics) => {
+      if (topics.length === 0 || !topics) {
+        const error = new Error("Not Found");
+        error.status = 404;
+        return next(error);
+      }
```

Comment:
The job of the controller is to get any parameters or queries from the request, call the model, and send the information to the client, so this type of logic would happen in the model instead.